### PR TITLE
Update proxmox.md

### PR DESCRIPTION
**Pull request by Mars-Sea**
Proxmox informations doesn't show #1406

### CODE REVIEW

**Comment by shamoon** [comment]
Yea, this is why I don't like to maintain this stuff in our docs. This was just updated https://github.com/benphelps/homepage-docs/pull/59

Maybe you and @shalak can figure it out, I don't even have a proxmox setup so I really cant comment

File: src/pages/en/services/proxmox.md [hunk 1]
```diff
@@ -28,6 +28,12 @@ You will need to generate an API Token for new or an existing user. Here is an e
     - User: user from bullet 8 above
     - Token ID: something informative like the application or purpose like `homepage`
     - Privilege Separation: Checked
+11. Click on the Permissions "folder"
```
Comment (shalak) [inline src/pages/en/services/proxmox.md:31]:
```suggestion
11. Go back to the "Permissions" menu
```

File: src/pages/en/services/proxmox.md [hunk 2]
```diff
@@ -28,6 +28,12 @@ You will need to generate an API Token for new or an existing user. Here is an e
...
+12. Click Add -> API Token Permission
+    - Path: /
+    - API Token: Your API
```
Comment (shalak) [inline src/pages/en/services/proxmox.md:34]:
```suggestion
    - API Token: select the Token ID created in Step 10
```

**Review by shalak — CHANGES_REQUESTED** [review]
A little cleanup suggestions, overall LGTM

If you choose to add the step 11, please remember to increment the subsequent ones ;)

**Review by Sharnobile — APPROVED** [review]

**Pull request merged**